frontend/src/App.jsx

```diff
@@ -20,13 +20,17 @@ function App() {
   }, [city]);
 
   const location = Geolocation();
-  if (location.loaded) {
-    axios
-      .get(
-        `https://api-adresse.data.gouv.fr/reverse/?lon=${location.coordinates.lng}&lat=${location.coordinates.lat}`
-      )
-      .then((response) => setCity(response.data.features["0"].properties.city));
-  }
+  React.useEffect(() => {
```

Comment:
You can't have multiple useEffect in one component, merge your code inside the other.

Reply:
I change some many thing like the navigation bar that integer a button that ask the location